### PR TITLE
[moby/v4.0.0] [TAR-622] Add Jenkinsfile

### PR DESCRIPTION
**Pull request by StefanScherer**
Hook up a Jenkinsfile to build the qemu binaries.

### CODE REVIEW

**Comment by justincormack** [comment]
Is this the right branch to PR this into? we arent actually building master.

**Comment by StefanScherer** [comment]
@justincormack My idea was to put it in moby/master first, then cherry-pick it to 3.1.0 and 4.0.0 as we do in other repos. Should I change the PR to another branch first?

I would also need some advice for the Jenkinsfile, probably it's too simple right now. 😅

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        git \
+        libtool \
+        libpixman-1-dev \
+        libglib2.0-dev \
+        pkg-config \
+        python
+
+WORKDIR /qemu
+
+COPY . /qemu
```
Comment (djs55) [inline Dockerfile:15]: I guess this is missing a download of the qemu source?

**Comment by tuonga** [comment]
> @justincormack My idea was to put it in moby/master first, then cherry-pick it to 3.1.0 and 4.0.0 as we do in other repos. Should I change the PR to another branch first?

My original thought was to keep the build system outside of this repo so you don't have to carry it from branch to branch, but this is OK too I guess.  Somebody just has to make sure to propagate changes to relevant branches.

We can probably blow away moby/v3.1.0 soon anyway.  Also I am not sure we'll ever use the -alpine branches - I didn't bother creating one for v4.0.0.

File: Dockerfile (hunk 1 above)
Comment (tuonga) [reply Dockerfile:15]: this is inside the qemu source.  It'd force us to carry any changes to the build from branch to branch though.

**Comment by StefanScherer** [comment]
Closing this in favor of https://github.com/docker/binfmt repo.

**Pull request closed without merging**